CMake fails in a rerun

CMake fails in the build directory when re-run after a change in the
source tree with the following message,

```
Re-run cmake file: Makefile older than: CMakeCache.txt
-- Found GPTL C: /home/jayeshkrishna/acme/pio/ParallelIO_install/lib/libgptl.a
CMake Error at src/clib/CMakeLists.txt:162 (check_type_size):
  Unknown CMake command "check_type_size".


-- Configuring incomplete, errors occurred!
See also "/home/jayeshkrishna/acme/pio/ParallelIO_build/CMakeFiles/CMakeOutput.log".
See also "/home/jayeshkrishna/acme/pio/ParallelIO_build/CMakeFiles/CMakeError.log".
Makefile:1257: recipe for target 'cmake_check_build_system' failed
make: *** [cmake_check_build_system] Error 1
/home/jayeshkrishna/utils/cmake-2.8.12.1/cmake-install/bin/cmake -H/home/jayeshkrishna/acme/pio/ParallelIO -B/home/jayeshkrishna/acme/pio/ParallelIO_build --check-build-system CMakeFiles/Makefile.cmake 0
Re-run cmake file: Makefile older than: CMakeCache.txt
-- Found GPTL C: /home/jayeshkrishna/acme/pio/ParallelIO_install/lib/libgptl.a
CMake Error at src/clib/CMakeLists.txt:162 (check_type_size):
  Unknown CMake command "check_type_size".


-- Configuring incomplete, errors occurred!
See also "/home/jayeshkrishna/acme/pio/ParallelIO_build/CMakeFiles/CMakeOutput.log".
See also "/home/jayeshkrishna/acme/pio/ParallelIO_build/CMakeFiles/CMakeError.log".
Makefile:1257: recipe for target 'cmake_check_build_system' failed
make: *** [cmake_check_build_system] Error 1
make: Nothing to be done for 'examples'.
/home/jayeshkrishna/utils/cmake-2.8.12.1/cmake-install/bin/cmake -H/home/jayeshkrishna/acme/pio/ParallelIO -B/home/jayeshkrishna/acme/pio/ParallelIO_build --check-build-system CMakeFiles/Makefile.cmake 0
Re-run cmake file: Makefile older than: CMakeCache.txt
-- Found GPTL C: /home/jayeshkrishna/acme/pio/ParallelIO_install/lib/libgptl.a
CMake Error at src/clib/CMakeLists.txt:162 (check_type_size):
  Unknown CMake command "check_type_size".


-- Configuring incomplete, errors occurred!

```

--- a/src/clib/CMakeLists.txt
+++ b/src/clib/CMakeLists.txt
@@ -159,6 +159,7 @@ if (NOT PnetCDF_C_FOUND AND NOT NetCDF_C_FOUND)
   message (FATAL_ERROR "Must have PnetCDF and/or NetCDF C libraries")
 endif ()
 
+include(CheckTypeSize)
 check_type_size("size_t" SIZEOF_SIZE_T)
 CHECK_TYPE_SIZE("long long" SIZEOF_LONG_LONG)
 if (NOT ${SIZEOF_SIZE_T} EQUAL ${SIZEOF_LONG_LONG})

